python/sglang/srt/model_executor/model_runner.py

```diff
@@ -195,9 +195,9 @@ def load_model(self):
             monkey_patch_vllm_qvk_linear_loader()
 
         self.dtype = self.vllm_model_config.dtype
-        if self.model_config.model_override_args is not None:
+        if self.model_config.model_overide_args is not None:
```

Comment:
This should be `model_override_args`; I tried compiling the engine with tp=8 and got

```
>>> from sglang import LLM, SamplingParams
>>> llm = LLM(model="meta-llama/Meta-Llama-3.1-405B-Instruct-FP8", tensor_parallel_size=8)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ubuntu/sglang/python/sglang/srt/serving/engine.py", line 190, in __init__
    self.llm_engine = Engine(engine_args)
  File "/home/ubuntu/sglang/python/sglang/srt/serving/engine.py", line 50, in __init__
    self.startup()
  File "/home/ubuntu/sglang/python/sglang/srt/serving/engine.py", line 89, in startup
    self.tokenizer_manager = TokenizerManager(self.engine_args)
  File "/home/ubuntu/sglang/python/sglang/srt/managers/tokenizer_manager.py", line 93, in __init__
    self.hf_config = get_config(
TypeError: get_config() got an unexpected keyword argument 'model_overide_args'
>>>
```